Sources/SwifterSwift/UIKit/UICollectionViewExtensions.swift

```diff
@@ -146,6 +146,14 @@ public extension UICollectionView {
 
         register(UINib(nibName: identifier, bundle: bundle), forCellWithReuseIdentifier: identifier)
     }
+    
+    /// SwifterSwift: Check whether IndexPath is valid within the CollectionView
+    ///
+    /// - Parameter indexPath: An IndexPath to check
+    /// - Returns: Boolean value for valid or invalid IndexPath
+    func isValidIndexPath(_ indexPath: IndexPath) -> Bool {
+        return indexPath.section < numberOfSections && indexPath.row < numberOfItems(inSection: indexPath.section)
```

Comment:
I know that nobody will create a index path with a negative index but should be a case we handle since we are validating it? It is not a request for change, just a question. What do you think?

Reply:
That's a good point, I'll update the code. Should I change the same method in the TableView extension?

Reply:
Sure, If you could also add the test cases for that would be great :))

Reply:
Done, check #696 🙃